README_duo.md

```diff
@@ -0,0 +1,19 @@
+TeraRanger Duo ROS Module
+=========================
+
+This is the ROS module for the TeraRanger Duo ranging sensor (www.teraranger.com).
+
+
+Using module
+============
+
+To use the ROS node you just need to:
+* Create a ROS Workspace
+* Copy the node terarangerduo package into the workspace src directory
+* Compile using: catkin_make 
+* Run using: rosrun terarangerduo terarangerduo_node _portname:=/dev/ttyUSB0
```

Comment:
This probably needs updating to reflect the new naming (we can also be make it a part of the main readme)